server.js

```diff
@@ -1,6 +1,7 @@
 require('dotenv').config()
 const { TOKEN, BOT_PREFIX } = process.env
 const fs = require('fs')
+const { exec } = require("child_process");
```

Comment:
Why do you need exec?

Reply:
Fragment from previous test - will remove now.